actors/evm/src/interpreter/instructions/context.rs

```diff
@@ -30,18 +30,18 @@ pub fn call_value<'r, BS: Blockstore, RT: Runtime<BS>>(
 
 #[inline]
 pub fn address<'r, BS: Blockstore, RT: Runtime<BS>>(
-    _state: &mut ExecutionState,
-    _platform: &'r System<'r, BS, RT>,
-) -> Result<(), StatusCode> {
-    todo!()
+    state: &mut ExecutionState,
+    platform: &'r System<'r, BS, RT>,
+) {
+    state.stack.push(U256::from(platform.rt.message().receiver().id().unwrap()))
 }
 
 #[inline]
 pub fn origin<'r, BS: Blockstore, RT: Runtime<BS>>(
     state: &mut ExecutionState,
     platform: &'r System<'r, BS, RT>,
 ) {
-    state.stack.push(U256::from(platform.rt.message().caller().id().unwrap()))
+    state.stack.push(U256::from(platform.rt.message().origin().id().unwrap()))
```

Comment:
Note that these will likely not be the final addresses we return here -- we might return Eth addresses, depending on where the addressing spec lands. But it's OK to get rolling for now.